OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -107,7 +107,7 @@ public ConnectionFailedLogic(Widget widget, OrderManager orderManager, NetworkCo
 			widget.Get<LabelWidget>("CONNECTING_DESC").GetText = () => $"Could not connect to {connection.Target}";
 
 			var connectionError = widget.Get<LabelWidget>("CONNECTION_ERROR");
-			connectionError.GetText = () => orderManager.ServerError ?? connection.ErrorMessage ?? "Unknown error";
+			connectionError.GetText = () => Ui.Translate(orderManager.ServerError) ?? connection.ErrorMessage ?? "Unknown error";
```

Comment:
Not translating `Unknown error`?

Reply:
That is not a server message. I will continue translating the UI when this gets merged.